### DISCUSSION TITLE
Airflow not Recognizing FAB_SECURITY_MANAGER_CLASS configuration

### DISCUSSION THREAD

**Discussion by dadonnelly316** [apache/airflow#34195] [General] [answered]
Airflow is not recognizing the custom security manager set in FAB_SECURITY_MANAGER_CLASS after upgrading to airflow 2.7.0.

We can tell our FAB_SECURITY_MANAGER_CLASS isn't being initialized because logging in the __init__ function is not logged when the webserver starts (and obviously our custom security manager wasn't being invoked.) 

Is anyone else having this issue? I cannot figure out the issue after spending all day debugging and reading fab/airflow documentation, but didn't want to open a ticket because I wasn't sure if this was specific to our deployment.

We were able to work around this by importing current_app from flask and setting current_app.config["SECURITY_MANAGER_CLASS"]=MyCustomSecurityManager within the global scope of webserver_config.py

**Comment by Taragolis** [comment, marked as answer]
https://github.com/apache/airflow/pull/33690 ?

**Comment by dadonnelly316** [comment]
ahh!. Thank you!
